install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -1,4 +1,13 @@
 {{- if and (.Values.agent) (not .Values.preflight.enabled) }}
+
+{{- /*  Default values with backwards compatibility */ -}}
```

Comment:
AFAIU, by 1.10 we will not be needing this? Maybe it's worth adding a deprecation comment for our future selfs?

Reply:
Having gone through this process for this PR, I don't think we have figured out a way to get rid of this option yet.